script/main.py

```diff
@@ -190,11 +190,30 @@ def init_db(conn):
     create_users_table(conn)    
   
   insert_users_to_db(conn, people)
-  
 
-def percentage():
-  print('A function summarizing the percentage of women / men in the database')
 
+def select_all_gender(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT gender FROM users")
+  gender_row = cur.fetchall()  
+  return gender_row
+
+
+def percentage(conn):
+  gender_row = select_all_gender(conn)
+  
+  male = 0
+  female = 0
+  for field in gender_row:
+    if field == ('male',):
+      male = male + 1
+    else:
+      female = female + 1
+  percentage_of_women = ((female * 100)/(female + male))
+  percentage_of_men = ((male * 100)/(female + male))
+  print(percentage_of_women, percentage_of_men)
+  return percentage_of_men, percentage_of_women
```

Comment:
If you print the result here. Then there is no reason to return any values from this function